kubernetes/gaffer-jhub/README.md

```diff
@@ -0,0 +1,13 @@
+# JupyterHub with Gaffer integrations
+
+This Helm Chart deploys Gaffer's [JupyterHub Options Server](../../docker/gaffer-jhub-options-server/) alongside the [JupyterHub](https://github.com/jupyterhub/zero-to-jupyterhub-k8s) Chart. The 'options server' works in cooperation with Jupyter Hub to allow pre-configured notebook instances to be provisioned for users so that they can easily interact with HDFS and Gaffer instances.
+
+This chart adds the following additional properties to [`jupyterhub.singleuser.profileList`](https://zero-to-jupyterhub.readthedocs.io/en/latest/resources/reference.html#singleuser-profilelist):
+
+| Property | Description | Default |
+|----------|-------------|---------|
+| `enable_hdfs` | If true, users will be prompted to select which HDFS instance they want to interact with when spawning a new notebook. The options server will provision a ConfigMap, that will be mounted into the notebook pod at `/etc/hadoop/conf`, containing Hadoop configuration files to connect to that instance by default. | `false` |
```

Comment:
I think the enable_* flags are set to true by default, aren't they?

Reply:
If the properties aren't provided for a particular profile then the default value is `false`.

One of the example profiles in `values.yaml` does list them and sets them to `true`, which might be why you think they are `true`?